.dockerignore

```diff
@@ -0,0 +1,10 @@
+node_modules
+npm-debug.log
+README.md
```

Comment:
Looks like you removed the README file altogether?  We should have one and translate it to rst (automated with pandoc).  Let me know if you want me to take care of that.